manifests/opa.yaml

```diff
@@ -152,7 +158,30 @@ spec:
         - name: opa-server
           secret:
             secretName: opa-server
-
+        - name: opa-config
+          configMap:
+            items:
+              - key: config.yaml
+                path: config.yaml
+            name: opa-config
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: opa-config
+  namespace: opa
+data:
+  config.yaml: |
+    services:
+      - name: discovery
+        url: https://discovery-opa-bundles.s3-eu-west-1.amazonaws.com
```

Comment:
This needs to be externalized into a variable e.g:

```yaml
opa:
   bundles:
      discovery:
           url: ""
           prefix: ""
           name: ""
```

Reply:
remediated,  thanks Mosh!